device-types/HPE/Aruba-IAP-505.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: HPE
+model: Aruba AP-505
+slug: hpe-aruba-ap-505
+part_number: AP-505
+u_height: 0
+is_full_depth: false
+comments: '[Aruba 500 Series Datasheet](https://www.arubanetworks.com/assets/ds/DS_AP500Series.pdf)'
```

Comment:
Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Reply:
Added